draft-ietf-oauth-resource-metadata.xml

```diff
@@ -243,26 +241,9 @@
 	    JSON array containing a list of the JWS <xref target="JWS" /> signing algorithms
 	    (<spanx style="verb">alg</spanx> values) <xref target="JWA" />
 	    supported by the protected resource
```

Comment:
There should be a `.` at the end of this sentence.

Reply:
This is not the end of the sentence.  The full sentence is:
> JSON array containing a list of the JWS <xref target="JWS" /> signing algorithms (<spanx style="verb">alg</spanx> values) <xref target="JWA" /> supported by the protected resource for signing resource responses, for instance, as described in <xref target="FAPI.MessageSigning"/>.

Reply:
now it's erroneously two sentences e3a57219cb1364185686d26ad820acece6095937

Reply:
Corrected